src/main/java/com/lightbend/akka/jpa/sample/User.java

```diff
@@ -0,0 +1,73 @@
+package com.lightbend.akka.jpa.sample;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.*;
+
+@Entity
```

Comment:
Maybe note here (as well) why it needs to be immutable, that it can't use joined collections etc. because it is sent across threads? At least I'd guess the whole attached magic lazy load/write back stuff to db breaks down in an async context.

Reply:
Totally agree. The pattern used here of separating read methods from write methods might be alien to typical JPA users, who are used to loading a bunch of objects, making a bunch of mutations to them, then saving the graph. It's one thing to show this way of doing it, but I think some explanation in comments would be helpful, too.